Makefile

```diff
@@ -74,6 +74,14 @@ docker-exe-%:
 docker-service-%:
 	$(MAKE) -C services/"$*" docker
 
+DOCKER_DEV_NETWORK := --net=host
+DOCKER_DEV_VOLUMES := -v `pwd`:/src/wire-server
+DOCKER_DEV_IMAGE   := wire/alpine-builder:local
+.PHONY: run-docker-dev
+run-docker-builder:
+	docker run -it $(DOCKER_DEV_NETWORK) $(DOCKER_DEV_VOLUMES) --rm $(DOCKER_DEV_IMAGE) /bin/bash || \
+	( echo "$(DOCKER_DEV_IMAGE) not found.  building locally.  hit ^C to interrupt." && make -C build/alpine builder )
```

Comment:
`echo "..." && make ...` && `docker run -it $(DOCKER_DEV_NETWORK) $(DOCKER_DEV_VOLUMES) --rm $(DOCKER_DEV_IMAGE) /bin/bash` (add the actual command one more time after being done building the alpine-builder ?)